src/module.c

```diff
@@ -12196,11 +12196,15 @@ int moduleLoad(const char *path, void **module_argv, int module_argc, int is_loa
     ValkeyModuleCtx ctx;
     moduleCreateContext(&ctx, NULL, VALKEYMODULE_CTX_TEMP_CLIENT); /* We pass NULL since we don't have a module yet. */
     if (onload((void *)&ctx, module_argv, module_argc) == VALKEYMODULE_ERR) {
-        serverLog(LL_WARNING, "Module %s initialization failed. Module not loaded", path);
         if (ctx.module) {
+            serverLog(LL_WARNING, "Module %s initialization failed. Module not loaded", path);
```

Comment:
Good but it is missing a "." in the end of the second sentence. Perhaps we can swap the sentences too? It seems more clear in the order: 1 what, 2 why.

```suggestion
            serverLog(LL_WARNING, "Module %s not loaded. Initialization failed.", path);
```

Reply:
yes, it is more clear in this way. But we have other places like this...

```
    if (onload == NULL) {
        dlclose(handle);
        serverLog(LL_WARNING,
                  "Module %s does not export ValkeyModule_OnLoad() or RedisModule_OnLoad() "
                  "symbol. Module not loaded.",
                  path);
        return C_ERR;
    }

        if (unload_status == VALKEYMODULE_ERR) {
            serverLog(LL_WARNING, "Module %s OnUnload failed. Unload canceled.", name);
            errno = ECANCELED;
            return C_ERR;
        }
```

Reply:
Ah, ok. Then just add the missing dot?

```suggestion
            serverLog(LL_WARNING, "Module %s initialization failed. Module not loaded.", path);
```